src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,76 @@
+package mate.jdbc.model;
+
+import java.util.List;
+
+public class Car {
+    private Long id;
+    private String model;
+    private String registrationNumber;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car(String model, String registrationNumber, Manufacturer manufacturer) {
+        this.model = model;
+        this.registrationNumber = registrationNumber;
+        this.manufacturer = manufacturer;
+    }
+
+    public Car(String model, String registrationNumber,
+               Manufacturer manufacturer, List<Driver> drivers) {
+        this.model = model;
+        this.registrationNumber = registrationNumber;
+        this.manufacturer = manufacturer;
+        this.drivers = drivers;
+    }
```

Comment:
Don't use this constructor. Use set to set drivers. In the constructor above initialize drivers = new ArrayList()

Reply:
ok